File: IMDB_group_1/Java Skeleton/src/main/java/imdb/entity/Film.java
package imdb.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "films")
public class Film {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private String genre;

    @Column(nullable = false)
    private String director;

    private int year;

    //default constructor
    public Film() {
    }

    public Film(String name, String genre, String director, int year) {
        this.name = name;
        this.genre = genre;
        this.director = director;
        this.year = year;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public String getDirector() {
        return director;
    }

    public void setDirector(String director) {
        this.director = director;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }
}

File: IMDB_group_1/JavaScript Skeleton/models/film.js
const Sequelize = require('sequelize');

module.exports = function(sequelize){

      const Film = sequelize.define("Film", {

          name: {
              type: Sequelize.TEXT,
              required: true,
              allowNull: false
          },

          genre: {
              type: Sequelize.TEXT,
              required: true,
              allowNull: false
          },

          director: {
              type: Sequelize.TEXT,
              required: true,
              allowNull: false
          },

          year: {
              type: Sequelize.INTEGER,
              required: true,
              allowNull: false
          }

      }, {
          timestamps: false
      });

      return Film;
};